app/views/provider/admin/account/users/index.html.slim

```diff
@@ -1,71 +1,65 @@
-- content_for :page_header_title, 'Users'
+- content_for :page_header_title, t('.page_header_title')
 
 - content_for :javascripts do
-  / HACK: temporary reuse css of existing pack "table_toolbar". Once toolbar is implemented,
-    import the whole pack with "javascript_packs_with_chunks_tag"
-  = stylesheet_packs_chunks_tag 'table_toolbar'
+  = javascript_packs_with_chunks_tag 'table_toolbar'
 
-table class="pf-c-table pf-m-grid-lg" role="grid" aria-label="Users table" id="users"
+table class="pf-c-table pf-m-grid-lg" role="grid" aria-label="Users table" data-toolbar-props=presenter.toolbar_props.to_json
   thead
     tr role="row"
       th role="columnheader" scope="col" Name
-      th role="columnheader" scope="col"
-        = current_user.field_label("email")
-      - if show_provider_sso_status_for_user?
+      th role="columnheader" scope="col" Email
+      - if presenter.show_provider_sso_status_for_user?(current_account)
         th role="columnheader" scope="col" Authenticated through SSO
       th role="columnheader" scope="col" Role
-      - if can?(:manage, :permissions)
-        th role="columnheader" scope="col" Permission Groups
-      th role="columnheader" scope="col" class="pf-c-table__action pf-m-fit-content"
-        - if can?(:create, Invitation) and can?(:see, :multiple_users)
-          = link_to_unless_current 'Invite a New User', new_provider_admin_account_invitation_path,
-                                                        class: 'action add'
+      - if presenter.show_permission_groups?
+        th role="columnheader" scope="col" Permission groups
+      td
 
   tbody role="rowgroup"
-    - @users.each do |user|
+    - presenter.users.each do |user|
+      - user = user.decorate
```

Comment:
Any reason you prefer to put this in the view rather than the presenter? I think you can call `.decorate` on the collection?...